modules/api/src/test/java/org/eclipse/xpanse/api/exceptions/handler/MonitoringExceptionHandlerTests.java

```diff
@@ -31,57 +29,66 @@
 import org.springframework.web.context.WebApplicationContext;
 
 @ExtendWith(SpringExtension.class)
-@ContextConfiguration(classes = {CredentialManageApi.class, CommonExceptionHandler.class,
-        CredentialCenter.class, MonitoringExceptionHandler.class, IdentityProviderManager.class})
+@ContextConfiguration(classes = {ServiceMetricsApi.class, MonitorMetricsService.class,
```

Comment:
Should MonitorMetricsService class be also renamed to just MetricsService?

Reply:
Ok